core/state_transition.go

```diff
@@ -76,10 +76,10 @@ type Message interface {
 }
 
 // IntrinsicGas computes the 'intrinsic gas' for a message with the given data.
-func IntrinsicGas(data []byte, contractCreation, isEIP155 bool, isEIP2028 bool) (uint64, error) {
+func IntrinsicGas(data []byte, contractCreation, isEIP2 bool, isEIP2028 bool) (uint64, error) {
```

Comment:
Interesting. Indeed, it's incorrect, but I'd prefer to use `isHomestead`, as that's what the caller-code uses, and we're using that name throughout. 
```suggestion
func IntrinsicGas(data []byte, contractCreation, isHomestead bool, isEIP2028 bool) (uint64, error) {
```

Reply:
https://github.com/ethereum/go-ethereum/pull/20300/commits/a323c4c7c036fc11ad3b67b3f79ca9fe4650cc46